docs/source/osds/scenarios.rst

```diff
@@ -205,6 +205,7 @@ The following keys are accepted for a ``filestore`` deployment:
 * ``data_vg`` (not required if ``data`` is a raw device or partition)
 * ``journal``
 * ``journal_vg`` (not required if ``journal`` is a partition and not a logical volume)
+* ``crush_device_class`` (not required, sets the crush device class for the OSD)
```

Comment:
the above mention "not required" because there is a condition: *not required if <reason>*, but that is not the case here. I think it would be fine to say *optional* instead